extensions/amp-inline-gallery/1.0/amp-inline-gallery-thumbnails.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {CSS as CAROUSEL_CSS} from '../../amp-base-carousel/1.0/base-carousel.jss';
+import {CSS as CAROUSEL_CSS} from '../../amp-base-carousel/1.0/component.jss';
```

Comment:
You'll have to update these paths in `dep-check-config.js` as well

Reply:
Thanks! Still getting in the habit of running `pr-check` before pushing

Reply:
Nah it's all good, I don't always do that either.